client/src/components/breadcrumbs.jsx

```diff
@@ -0,0 +1,37 @@
+//import { useState, useEffect } from 'react'
+//import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+// internal logic for breadcrumbs:
+// "/"   -->  ["home"]
+// "/one" --> ["home", "one"]
+// "/one/one_one" --> ["home", "one", "one_one"] --> home / one / one_one
+//
+// hint:
+// ["home", "one", "one_one"].join("/")
+
+const Breadcrumbs = ({ currentPath, fetchFiles }) => {
+  const pathnames = currentPath === '/' ? [] : currentPath.split('/').slice(1)
+
+  return (
+    <nav aria-label='Breadcrumb' className='breadcrumbs'>
+      <ul className='breadcrumb'>
+        <li className='breadcrumb-item' onClick={() => fetchFiles('/')}>
+          home
+        </li>
+        {currentPath !== '/' &&
+          pathnames.map((name, index) => {

```

Comment:
you don't need `pathnames` now.
please use that logic in line # 14 for `currentPath` here